icu4j/pom.xml

```diff
@@ -328,6 +331,61 @@
       </build>
     </profile>
 
+    <!-- This is used to gather API data used to generate API reports
+      (regular, draft, deprecated, status consistency, etc.)
+      To run it:
+        mvn install -DskipITs -DskipTests
+        mvn site -DskipITs -DskipTests -P gatherapi
+    -->
+    <profile>
+      <id>gatherapi</id>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-javadoc-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>aggregate</id>
+                <phase>site</phase>
+                <goals>
+                  <goal>aggregate</goal>
+                </goals>
+              </execution>
+            </executions>
+            <configuration>
+              <!--
+                The way "gatherapi" works is that com.ibm.icu.dev.tool.docs.GatherAPIData is a doclet.
+                It completely replaces the standard javadoc doclet.
+                It is invoked for each class and method, but instead of collecting the comments
+                and generating html, it only adds the method names to the API report.
+
+                See https://docs.oracle.com/javase/8/docs/technotes/guides/javadoc/doclet/overview.html
+
+                maven-javadoc-plugin fails because it does not detect the generated javadoc
+                And it does not detect it because it does not exist, as expected (it is not generated).
+                So we suppress that faulure, bacuse it works as intended.
```

Comment:
```suggestion
                maven-javadoc-plugin fails because it does not detect the generated javadoc.
                And it does not detect it because the javadoc does not exist, as expected (it is not generated).
                So we suppress that failure, because "gatherapi" works as intended.
```